python/sdk/merlin/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = "0.12.0"
+VERSION = "0.12.1"
```

Comment:
You also need to update this line: https://github.com/gojek/merlin/blob/main/python/pyfunc-server/requirements.txt#L9

Reply:
since the version that used by pyfunc-server always the latest, I will remove specific version on the requirements.txt 
so the requirements look like this `merlin-sdk` only, wdyt

Reply:
Agree with that. I like that approach!